site/src/documents/pages/blocks/FooterContentBlock.tsx

```diff
@@ -0,0 +1,138 @@
+import { PropsWithData, withPreview } from "@comet/cms-site";
+import { FooterContentBlockData } from "@src/blocks.generated";
+import { DamImageBlock } from "@src/common/blocks/DamImageBlock";
+import { LinkBlock } from "@src/common/blocks/LinkBlock";
+import { RichTextBlock } from "@src/common/blocks/RichTextBlock";
+import { Typography } from "@src/common/components/Typography";
+import * as React from "react";
+import styled from "styled-components";
+
+export const FooterContentBlock = withPreview(
+    ({ data: { text, image, linkList, copyrightNotice } }: PropsWithData<FooterContentBlockData>) => {
+        return (
+            <Root>
+                <ImageTextWrapper>
+                    <ImageWrapper>
+                        <DamImageBlock data={image} aspectRatio="1/1" objectFit="contain" />
+                    </ImageWrapper>
+                    <RichTextWrapper>
+                        <RichTextBlock data={text} />
+                    </RichTextWrapper>
+                </ImageTextWrapper>
+                <HorizontalLine />
+                <LinkCopyrightWrapper>
+                    {!!linkList.blocks.length && (
+                        <LinksBlock>
+                            {linkList.blocks.map((block) => (
+                                <LinkText as={LinkBlock} key={block.key} data={block.props.link} variant="p200">
+                                    {block.props.text}
+                                </LinkText>
+                            ))}
+                        </LinksBlock>
+                    )}
+                    {copyrightNotice && <CopyrightNotice variant="p200">{copyrightNotice}</CopyrightNotice>}
+                </LinkCopyrightWrapper>
+            </Root>
+        );
+    },
+    { label: "Footer" },
+);
+
+const Root = styled.footer`
+    margin-top: auto;
+    background-color: ${({ theme }) => theme.palette.grey["900"]};
+    color: ${({ theme }) => theme.palette.grey["50"]};
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 60px 16px;
+
+    ${({ theme }) => theme.breakpoints.sm.mediaQuery} {
+        padding: 60px 30px;
+    }
+
+    ${({ theme }) => theme.breakpoints.md.mediaQuery} {
+        padding: 60px 45px;
+        flex-direction: row;
+        justify-content: space-between;
+    }
+
+    ${({ theme }) => theme.breakpoints.lg.mediaQuery} {
+        position: relative;
+    }
+`;
+
+const ImageTextWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 30px;
```

Comment:
```suggestion
    gap: ${({ theme }) => theme.spacing.S600};
```

here also? try to use as much as possible from the spacings please.